TestResultSummaryService/parsers/BenchmarkParser.js

```diff
@@ -155,9 +141,18 @@ class BenchmarkParser extends Parser {
         } else {
             buildResult = "FAILURE";
         }
+        const javaVersion = this.exactJavaVersion( output );
+        let jdkDate = null;
+        let curRegexResult = null;
+        // parse jdk date from javaVersion
+        if ( ( curRegexResult = javaBuildDateRegex.exec( javaVersion ) ) !== null ) {
+                jdkDate = curRegexResult[1];
+        }
```

Comment:
Please put the line 145 - line 150 to `exactJavaVersion `

Reply:
Updated as required!